Invoke-FslShrinkDisk.ps1

```diff
@@ -1434,8 +1439,6 @@ function Write-VhdOutput {
             }
             Start-Sleep 1
         }
-
```

Comment:
this is not the information that i have in the original ps1 script.  Am i missing something?